Sources/AblyAssetTrackingSubscriber/WorkerQueue/SubscriberWorkerFactory.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import AblyAssetTrackingInternal
+
+internal class SubscriberWorkerFactory: WorkerFactory
+{
+    public typealias PropertiesType = SubscriberWorkerQueueProperties
```

Comment:
I don't think these `typealias` are necessary – the compiler should infer them from the signature of `createWorker`.

Reply:
86abd82